.github/workflows/cypress-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: "dhis2: cypress tests"
+
+on: [push]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          start: 'yarn start'
+          wait-on: 'http://localhost:3000'
+          wait-on-timeout: 60
+
+      - name: Upload artifacts
+        if: failure()
+        uses: actions/upload-artifact@v1
+        with:
+          name: failed-screenshots
+          path: cypress/screenshots
+    env:
+      CI: true
+      REACT_APP_DHIS2_BASE_URL: 'https://debug.dhis2.org/2.34dev/'
```

Comment:
here we pass the environment variables for the test. 


Phil explained how to set up an instance with some seeded data. The plan here would be to set up an instance just for us that we would call somehow `https://capture-app-test.dhis2.org/2.34dev/`